src/cloud-api-adaptor/test/e2e/aws_test.go

```diff
@@ -101,13 +101,19 @@ func TestAwsDeletePod(t *testing.T) {
 }
 
 func TestAwsCreateNginxDeployment(t *testing.T) {
+	t.Skip("Test not passing")
```

Comment:
Hmm, isn't this a basic test? Any idea why is it failing ?

Reply:
Hi @bpradipt !

The nginx test was unstable on libvirt while ago (https://github.com/confidential-containers/cloud-api-adaptor/issues/2046) but then got re-enabled recently. Interestingly that I tried to reproduce the fails locally many time but I never caught it, it seems something related with CI environment. As for AWS, I didn't have time to debug it yet....

Reply:
Thanks @wainersm. I was just curious :-) ..